### PR TITLE
start siot on port 80

### PR DESCRIPTION
**Pull request by cbrake**

### CODE REVIEW

File: recipes-siot/simpleiot/simpleiot/siot [hunk 1]
```diff
@@ -13,8 +13,8 @@ mkdir -p $DATADIR
 case "$1" in
   start)
         echo -n "Starting $DESC"
-         check_dbs
-        start-stop-daemon -S -m -p $PIDFILE -b -x /usr/bin/siot
+        check_dbs
+        start-stop-daemon -S -m -p $PIDFILE -b -x /bin/sh -- -c 'SIOT_PORT=80 /usr/bin/siot'
```
Comment (kraj) [inline recipes-siot/simpleiot/simpleiot/siot]: can we try
`start-stop-daemon -S -m -p $PIDFILE -b -x /usr/bin/env  SIOT_PORT=80 /usr/bin/siot'`

Comment (cbrake) [reply recipes-siot/simpleiot/simpleiot/siot]: updated PR to use /usr/bin/env -- agreed, that is probably a little cleaner

Comment (cbrake) [reply recipes-siot/simpleiot/simpleiot/siot]: ok, I think I got it now

**Review by kraj — APPROVED** [review]
LGTM

**Pull request merged**